first_order_model/train.py

```diff
@@ -272,6 +380,8 @@ def train(config, generator, discriminator, kp_detector, checkpoint, log_dir, da
            
             # record a standard set of metrics
             if metrics_dataloader is not None:
+                for x in metrics_dataloader:
```

Comment:
What's the purpose here? This will break existing compatibility with my training pipeline, and should be avoided at all costs.